docs/admin/CertificateTransparency.adoc

```diff
@@ -0,0 +1,114 @@
+
+== Configuration
+
+The configuration for CT is in the CA's CS.cfg (/var/lib/pki/<instance name>/ca/conf/CS.cfg).
+
+=== ca.certTransparency.mode
+
+ca.certTransparency.mode specifies CT mode. There are three Certificate Transparency modes:
+
+* *_disabled_*: issued certs will not carry SCT extension
+* *_enabled_*: issued certs will carry SCT extension
+* *_perProfile_*: certs enrolled through those profiles that contain the following policyset will carry SCT extension: SignedCertificateTimestampListExtDefaultImpl
+
+Default is _enabled_
+
+=== ca.certTransparency.log.num
+
+ca.certTransparency.log.num specifies the total number of CT logs defined in the configuration.  Not all defined CT log entries are considered active.  See <<ca.certTransparency.log.<id>.enable>>.
+
+=== ca.certTransparency.log.<id>.*
+
+ca.certTransparency.log.<id>.* specifies information pertaining to the log <id>, where <id> is a unique id you assign to the CT log server to differentiate it from other CT logs.
+
+The parameter names follow each _ca.certTransparency.log.<id>._ belongs to the _<id>_. and they are specified below:
+
+ca.certTransparency.log.<id>.enable:: specifies whether the <id> CT log is enabled (_true_) or disabled (_false_).
+
+ca.certTransparency.log.<id>.pubKey:: contains the base64 encoding of the CT log's public key
+
+ca.certTransparency.log.<id>.url:: contains the base64 encoding of the CT log's url.
+
+ca.certTransparency.log.<id>.version:: specifies the CT version number that the CT supports (as well as the CT log server);  It currently only supports version 1.
+
+== Example / Test
+
+The following is an actual test against Google CT test logs, which serves as an exmaple on how to test a setup.
+A more comprehensive test procedure would involve setting up a TLS server and test for its cert's inclusion from it's specified CT logs.
+However, the following serves as a quick test that checks for inclusion of the SCT extension once a certificate has been issued.
+
+=== CT Test Configuration in CS.cfg
+
+[literal]
+ca.certTransparency.mode=enabled
+ca.certTransparency.log.1.enable=true
+ca.certTransparency.log.1.pubKey=MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEw8i8S7qiGEs9NXv0ZJFh6uuOm<snip>
+ca.certTransparency.log.2.enable=true
+ca.certTransparency.log.2.pubKey=MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEKATl2B3SAbxyzGOfNRB+AytNTG<snip>
+ca.certTransparency.log.2.url=http://ct.googleapis.com:80/logs/crucible/
+ca.certTransparency.log.2.version=1
+ca.certTransparency.log.3.enable=false
+ca.certTransparency.log.3.pubKey=MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEiKfWtuoWCPMEzSKySjMjXpo38W<snip>
+ca.certTransparency.log.3.url=http://ct.googleapis.com:80/logs/solera2020/
+ca.certTransparency.log.3.version=1
+ca.certTransparency.log.num=3
+
+. Generate a CSR: e.g.
+.. _PKCS10Client -d . -p passwd -l 2048 -n "cn=ladycfu.test.redhat.com,OU=ladycfu-TEST,O=TestDomain" -o pkcs10-TLS.req_
```

Comment:
I'd suggest to use code block for command lines to make it more legible, and you can break long lines like this:
```
PKCS10Client -d . -p passwd -l 2048 \
    -n "cn=ladycfu.test.redhat.com,OU=ladycfu-TEST,O=TestDomain" \
    -o pkcs10-TLS.req
```
This will make it easier to copy & paste the command into an actual terminal.